src/test/services/mocks/MockFAQService.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { API_ROOT } from "../../../api-config";
+
+const mockFAQ = (s) => jest.fn().mockImplementation((url, config) => {
+    if(!config) {
+      if (url.indexOf(`${API_ROOT}/api/faqs`) != -1) {
```

Comment:
!= should be !==, oh javascript 😄

Reply:
or you could just do `url.includes('/faqs')`

Reply:
we have some of this in https://github.com/liamweldon/cs4500_sp19_client_react_cccp/blob/master/src/test/services/FAQService.test.js i guess i did it before someone started the pattern pattern of the mock folder